packages/gluestick/src/cli/index.js

```diff
@@ -56,18 +56,6 @@ commander.option('-l, --light', 'use light color schema for logging', () => {
 });
 
 commander
-  .command('new')
```

Comment:
Why `new` command is removed?

Reply:
I wasn't 100% certain that it should go, so correct me if I'm wrong about any of this.  The biggest reason I removed it was that it exists in `package/gluestick-cli`.  I guessed that it has been brought along during updates as an oversight instead of deliberately. 

I looked at the commit history to try to figure out whether my guess was right or not.  This one hasn't received all the updates as the cli version, so they're mismatched.  The updates it has received seem to be more cleanup than functional.

That said, I could understand if there's a reason to have both version of the command, so again, tell me if I'm missing it.

Reply:
It is essential to have this command here. The `new` from `gluestick-cli` creates `package.json` and installs dependencies then it spawns `new ` from `gluestick` to create project files.

Reply:
Ah ok, I definitely should have tried to run `gluestick new`.  Then it would have been obvious I broke it.

I should have mentioned in last comment: this command became obvious when I started iterating over the local commander instance.  I thought it shouldn't be here because it's not part of the cli-api.  No user would ever call this command directly, right?

If so, I might migrate this command elsewhere and update the global's spawned command.  For example, move it to a new command in `packages/gluestick/bin`.  That would allow reflecting on the main commander instance without special filtering, and this file would be readable as a description of the cli interface.  What do you think?

Reply:
I renamed the command to `_init` and added the no-help commander option to conceal it.  Then I added a filter of the commander settings to exclude commands with a leading '_'.

I tried to test that `gluestick new` was still functional, and I think I succeeded, but it took a lot of linking between projects.  I had to link global gluestick to my working branch (no problem), and then create a project.  I used the `--dev` option, which would fail because it couldn't find the generator template.  Then I linked the project's gluestick dependency to my checkouts `/packages/gluestick`, and installed that and linked it to my checkout's `packages/gluestick-generators`.  After that I was able to execute `gluestick _init` and completion the project creation.

I'm hoping that most of the difficulty vanishes once the change is deployed to npm and all the packages are in the correct location.

Reply:
This might break project generation if user haven't updated `gluestick-cli` and ppl don't do it often. I consider it a breaking change, but I want @toddw opinion about this.

Reply:
That makes sense.  I had imagined that scenario to be impossible because a user couldn't be running the completion without updating `gluestick-cli`, but this change would affect local gluestick without forcing a `-cli` update.  So failure scenario is:
1. previous version of `gluestick-cli`, command `gluestick new` installs updated version of `gluestick`
2. Updated package does not provides the `_init` command, not `new`.
3. Previous `-cli` fails attempting to spawn `new`, so the project generator never runs.

I don't see any way to avoid backing out this change.  I tried to look at https://github.com/TrueCar/gluestick/pull/1097/files#diff-01fb1ca83ea3c067c9cc6c70105fb1f7R102 to see if there was some recourse there based on how local `new` is called, but that would still require an updated `gluestick-cli`.  I can still add `noHelp`, and I make my commander inspection filter be explicit.

Out of curiosity: this problem won't be solved by semantic versioning.  Users could always have stale `gluestick-cli`, even after we publish 2.0.  What's the strategy we will apply when that happens?  And...any chance we could apply it right now?  :)

Reply:
Is there no way to make it so tab completion simply does nothing in the old version but does work in the new version? Does this have to be a breaking change?

Reply:
No, totally not required.  I can back it out.  I thought I had a thing that should be changed, but the breaking isn't worth the benefit.